lib/adapters/mongodb/converters.js

```diff
@@ -51,7 +51,11 @@ module.exports = function() {
                 Hoek.assert(val.isJoi, 'attribute values in the hh schema should be defined with Joi')
                 return schemaMap[val._type]
             })
-    
+        mongooseSchema.relationships =
+            _.mapValues(hhSchema.relationships, function (val, key) {
+                return mongoose.Schema.Types.Mixed
```

Comment:
I know I told you to use ```Mixed``` here, but if the structure of ```relationship``` is know, can we not treat it like another Mongoose schema and reference it from here?